### PR TITLE
feat(climate): Support more horizontal swing

### LINKED ISSUE #52: 如何控制冷氣導風板方向
想要可以控制冷氣導風板往左或往右固定，請問怎麼控制？

### PR DESCRIPTION
**Pull request by sykuang**
fixes #52

Support following customized swing mode

- "Horizontal Leftmost"
- "Horizontal Middle Left"
- "Horizontal Central"
- "Horizontal Middle Right"
- "Horizontal Rightmost"
- "Horizontal Leftmost + Vertical Swing"
- "Horizontal Middle Left + Vertical Swing"
- "Horizontal Central + Vertical Swing"
- "Horizontal Middle Right + Vertical Swing"
- "Horizontal Rightmost + Vertical Swing"

### CODE REVIEW

File: custom_components/jcihitachi_tw/climate.py [hunk 1]
```diff
@@ -210,14 +232,33 @@ def fan_modes(self):
...
+                    return SWING_HORIZONTAL_LEFTMOST_SWING
+                elif status.horizontal_wind_direction == "middleleft":
+                    return SWING_HORIZONTAL_MIDDLE_LEFT_SWING
+                elif status.horizontal_wind_direction == "central":
+                    return SWING_HORIZONTAL_CENTRAL_SWING
+                elif status.horizontal_wind_direction == "middleright":
+                    return SWING_HORIZONTAL_MIDDLE_RIGHT_SWING
+                elif status.horizontal_wind_direction == "rightmost":
+                    return SWING_HORIZONTAL_RIGHTMOST_SWING
                 return SWING_VERTICAL
```
Comment (qqaatw) [inline custom_components/jcihitachi_tw/climate.py:248]: will this line ever be reached?

File: custom_components/jcihitachi_tw/climate.py [hunk 2]
```diff
@@ -32,6 +32,17 @@
 PRESET_MOLD_PREVENTION = "Mold Prev"
 PRESET_ECO_MOLD_PREVENTION = "Eco & Mold Prev"
 
+SWING_HORIZONTAL_LEFTMOST = "Horizontal Leftmost"
+SWING_HORIZONTAL_MIDDLE_LEFT = "Horizontal Middle Left"
+SWING_HORIZONTAL_CENTRAL = "Horizontal Central"
+SWING_HORIZONTAL_MIDDLE_RIGHT = "Horizontal Middle Right"
+SWING_HORIZONTAL_RIGHTMOST = "Horizontal Rightmost"
+SWING_HORIZONTAL_LEFTMOST_SWING = "Horizontal Leftmost + Vertical Swing"
```
Comment (qqaatw) [inline custom_components/jcihitachi_tw/climate.py:40]:
```suggestion
SWING_HORIZONTAL_LEFTMOST_VERTICAL_SWING = "Horizontal Leftmost + Vertical Swing"
```

File: custom_components/jcihitachi_tw/climate.py [hunk 3]
```diff
@@ -32,6 +32,17 @@
...
+SWING_HORIZONTAL_MIDDLE_LEFT_SWING = "Horizontal Middle Left + Vertical Swing"
+SWING_HORIZONTAL_CENTRAL_SWING = "Horizontal Central + Vertical Swing"
+SWING_HORIZONTAL_MIDDLE_RIGHT_SWING = "Horizontal Middle Right + Vertical Swing"
+SWING_HORIZONTAL_RIGHTMOST_SWING = "Horizontal Rightmost + Vertical Swing"
```
Comment (qqaatw) [inline custom_components/jcihitachi_tw/climate.py:44]: Ditto.

**Review by qqaatw — COMMENTED** [review]
Thanks for your contribution. Does it work locally on your end?

I don't have a Hitachi device that equips this feature so I can't test it myself. If someone reports issues with this change the PR may get reverted, sorry.

File: custom_components/jcihitachi_tw/climate.py (hunk 1 above)
Comment (qqaatw) [reply custom_components/jcihitachi_tw/climate.py:248]: If never, please remove it and log an error here.

File: custom_components/jcihitachi_tw/climate.py (hunk 2 above)
Comment (sykuang) [reply custom_components/jcihitachi_tw/climate.py]: Submit new version

File: custom_components/jcihitachi_tw/climate.py (hunk 3 above)
Comment (sykuang) [reply custom_components/jcihitachi_tw/climate.py]: Remove central and central swing.

**Comment by sykuang** [comment]
> Thanks for your contribution. Does it work locally on your end?
> 
> I don't have a Hitachi device that equips this feature so I can't test it myself. If someone reports issues with this change the PR may get reverted, sorry.

I conducted a test on my air conditioner at home and it appears to be functioning properly.

**Review by qqaatw — APPROVED** [review]

**Pull request merged**